src/components/ComparisonTable.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from "react";
+import DownloadsGraph from "./DownloadsGraph";
+import { TPackage } from "../global/types";
+import { TRows } from "../global/types";
+import { fetchData } from "../utils/apiUtils";
+import { rowFormatter } from "../utils/rowFormatter";
+import TableComponent from "./TableComponent";
+
+export default function ComparisonTable({ selectedPackages }) {
+  const [firstPackageData, setfirstPackageData] = useState<
+    TPackage | undefined
+  >();
+  const [secondPackageData, setSecondPackageData] = useState<
+    TPackage | undefined
+  >();
+  const [rows, setRows] = useState<Array<TRows>>([]);
+
+  useEffect(() => {
+    const promiseOne = fetchData(
```

Comment:
promise one and two are not specifying what data we are receiving. Try to rename the variables relative to their use.